polaris-react/src/components/Select/Select.scss

```diff
@@ -199,32 +166,20 @@
   // so that errors still have red borders.
   // stylelint-disable-next-line selector-max-combinators, selector-max-specificity, selector-max-class -- generated by polaris-migrator DO NOT COPY
   .Input:focus-visible ~ .Backdrop {
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    @include focus-ring($style: 'focused');
-
-    #{$se23} & {
```

Comment:
Interesting that there's not a `@include no-focus-ring;` to remove here. At first I though that meant the `@include focus-ring($style: 'focused');` should remain, but looking at the later code, a more specific selector which _does_ include `@include no-focus-ring;` wins:

```css
.error .Input:focus-visible ~ .Backdrop {
  @include focus-ring($style: 'focused');
}
/* vs ... */
html[class~="Polaris-Summer-Editions-2023"] .Input:focus-visible ~ .Backdrop {
  @include focus-ring($style: 'focused');
  @include no-focus-ring;                /* <--- WINNER */
}
```